pastas/decorators.py

```diff
@@ -3,6 +3,13 @@
 
 logger = getLogger(__name__)
 
+USE_NUMBA = True
+
+
+def set_use_numba(b: bool) -> None:
+    global USE_NUMBA
+    USE_NUMBA = b
+
```

Comment:
Should this not be in utils.py ? It is not really a decorators. But I can see why you put it here..

Reply:
I agree that it's in a bit of an awkward place but let's keep it here. I think that it is most logical to keep it close to the njit decorator.